posthog/caching/warming.py

```diff
@@ -37,16 +40,48 @@
 LAST_VIEWED_THRESHOLD = timedelta(days=7)
 
 
-def priority_insights(team: Team, shared_only: bool = False) -> Generator[tuple[int, Optional[int]], None, None]:
+def teams_enabled_for_cache_warming() -> list[int]:
+    all_teams = Team.objects.all()
+    enabled_team_ids = []
+
+    for team in all_teams:
```

Comment:
Are we sure this isn't a performance issue? We're looping over all teams after all.

Don't know if this can be done in another way with feature flags though. I imagine there might be a way to retrieve the feature flag config and get the team ids from that.

Reply:
As discussed, this is what feature flags team recommended :) 
Also we've got `only_evaluate_locally=True,` which will ensure it doesn't make API calls. Tested this on prod console and it only takes a couple of seconds